packages/desktop-client/src/components/common/Button.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import { css } from 'glamor';
+import type { CSSProperties } from 'glamor';
+
+import { styles, colors } from '../../style';
+
+interface ButtonProps extends React.HTMLProps<HTMLButtonElement> {
+  pressed?: boolean;
+  primary?: boolean;
+  hover?: boolean;
+  bare?: boolean;
+  disabled?: boolean;
+  hoveredStyle?: CSSProperties;
+  activeStyle?: CSSProperties;
+  bounce?: boolean;
+  as?: 'button';
```

Comment:
Could this ever be something else?

Reply:
Yea, for sure. But I'm not yet sure what.. We will definitely want to update this type as we discover more existing use-cases. But until then I'd suggest keeping the scope as small as possible.